dat/events/dvaered/warlords_battle.lua

```diff
@@ -14,9 +14,9 @@
 --  A battle between two Dvaered warlords. The player can join one of them and get a reward
 
 require "fleethelper"
-require "fleet_form"
 require "proximity"
 require "numstring"
+local formation = require "scripts/formation"
```

Comment:
you shouldn't need to prefix `scripts/` as it should be in the module loading path